.github/workflows/main.yml

```
@@ -33,18 +33,16 @@ jobs:
 
       envs: |
         # Standard tests
-        - linux: py38-test
-        - linux: py39-test
+        - linux: py311-test
```

```suggestion
        - linux: py310-test
```
should keep at least one 3.10 job as we no longer can run it for dev

---

Whoops, good call. Thought I kept one.